build-repo

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ "$#" -ne 1 ]; then
+  echo -e "Please provide repo package name to build\nUsage: docker run --rm -v \$(pwd):/pkg maximbaz/arch-build-aur /bin/bash -c '/build-repo <package-name>'"
+  exit 1
+fi
+
+package=$1
+
+chown -R makepkg:users /build
+
+cd /tmp && sudo -u makepkg asp checkout $package && mv $package/trunk/* /build && cd /
```

Comment:
I would prefer to split one command per line instead of `&&`. And replace `trunk` with `repos/*/` if you agree :)

Reply:
More than happy to split one command per line.

Re the trunk issue, building the repo version was my first thought as well, but what if a package is in more than one repo, probably because of testing?
I guess there could be a few lines of code to check if there's more than one subdirectory, and if so pick the one that does not contain `testing` - what do you think? Is there any scenario where that would break too?

Reply:
Ooh yes you are totally right... Besides `testing` there also are `unstable` and `staging`. I think we better whitelist the alternatives we want to support...

Maybe this? 

```
mv "$package"/repos/{core,extra,community}-{any,x86_64}/* build/
```

Reply:
hey @argymeg, a friendly ping 😉 let me know your thoughts, also if don't have time to look into this, I'll then try to finish it myself 👍 but I'm okay to wait and generally prefer you to finish because its important that this functionality then works for your use case 😉

Reply:
Gah, thanks for the ping, completely fell off my radar there! I'll look into it ASAP.